client/src/Routes.js

```diff
@@ -0,0 +1,206 @@
+            <Route path="login/:email?" element={<Login />} />
+            <Route path="forgotpassword/:email?" element={<ForgotPassword />} />
+            <Route
+              path="resetpasswordemailsent/:email?"
+              element={<ResetPasswordEmailSent />}
+            />
+            <Route path="resetPassword/:token" element={<ResetPassword />} />
+
+            <Route
+              path="organizationedit/:id?"
+              element={<OrganizationEdit />}
+            />
+            <Route path="muidemo" element={<MuiDemo />} />
+            <Route
+              path="verificationdashboard"
+              element={
+                <PrivateRoute
+                  roles={["isAdmin", "isDataEntry", "isCoordinator"]}
+                >
+                  <VerificationDashboard />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="verificationadmin"
+              element={
+                <PrivateRoute roles={["isAdmin", "isCoordinator"]}>
+                  <VerificationAdmin />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="parentorganizations"
+              element={
+                <PrivateRoute roles={["isAdmin"]}>
+                  <ParentOrganizations />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="tags"
+              element={
+                <PrivateRoute roles={["isAdmin"]}>
+                  <TagAdmin />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="suggestions"
+              element={
+                <PrivateRoute roles={["isAdmin"]}>
+                  <Suggestions />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="logins"
+              roles={["isAdmin", "isCoordinator"]}
+              element={
+                <PrivateRoute roles={["isAdmin", "isCoordinator"]}>
+                  <Logins />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="features"
+              element={
+                <PrivateRoute roles={["isAdmin"]}>
+                  <Features />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="securityadmindashboard"
+              element={
+                <PrivateRoute roles={["isGlobalAdmin", "isSecurityAdmin"]}>
+                  <SecurityAdminDashboard />
+                </PrivateRoute>
+              }
+            />
+            <Route
+              path="organizationimport"
+              element={
+                <PrivateRoute roles={["isAdmin"]}>
+                  <ImportFile />
+                </PrivateRoute>
+              }
+            />
+          </Route>
+
+          <Route exact path="fallback" element={<Fallback />} />
```

Comment:
To handle undefined admin paths, add a catch-all route below `organizationimport` route, redirecting to the fallback route:

```    
// Below `organizationimport` route, add:
   <Route path="*" element={<Navigate to="/fallback" replace />} />
</Route> // Closes admin routes
// Place the fallback route outside the admin routes
   <Route exact path="fallback" element={<Fallback />} />
```